cmd/utils/flags.go

```diff
@@ -117,6 +118,22 @@ var (
 		Name:  "override.prague",
 		Usage: "Manually specify the Prague fork time, overriding the bundled setting",
 	}
+	OverrideNormalcyBlockFlag = flags.BigFlag{
+		Name:  "override.normalcyblock",
+		Usage: "Manually specify the NormalcyBlock fork block, overriding the bundled setting",
+	}
+	OverrideShanghaiTimeFlag = flags.BigFlag{
+		Name:  "override.shanghaitime",
+		Usage: "Manually specify the time for shanghai fork",
+	}
+	OverrideLondonBlockFlag = flags.BigFlag{
+		Name:  "override.londonblock",
+		Usage: "Manually specify the block for london fork",
+	}
```

Comment:
- note - these are already in zkevm now so should be fixed with a rebase / merge in